### ISSUE TITLE
Suggestion: 16 folder-pictures and image-x-generic icons

### ISSUE THREAD

**Issue by malysps** [shimmerproject/elementary-xfce#102]
Right now `/elementary-xfce/places/16/folder-pictures.svg` icon is a symlink to `/mimes/16/image-x-generic.svg` and it looks like this ![image-x-generic](https://user-images.githubusercontent.com/16068365/27488322-f7732a2a-5836-11e7-98dc-6d3b050b7969.png) In upstream (https://github.com/elementary/icons) it isn't a symlink and `folder-pictures.svg` icon looks like this ![folder-pictures-e](https://user-images.githubusercontent.com/16068365/27488396-3c12ef94-5837-11e7-83b6-d6224c5dc23d.png) and I think it's a lot nicer and more recognizable. Additionally, in upstream elementary icon `/mimes/16/image-x-generic.svg` now looks like this ![image-x-generic-e](https://user-images.githubusercontent.com/16068365/27488463-691ae83e-5837-11e7-8000-7e24f4b20e01.png)

So to summarize:
- replace `/places/16/folder-pictures.svg` symlink with proper icon from upstream
- replace `/mimes/16/image-x-generic.svg` icon with new version from upstream

**Comment by ochosi** [comment]
The main point for integrating this (as well as any other change from upstream) is whether both icons are available in all sizes that we currently ship in elementary-xfce and if not the task is to draw the remaining sizes.

**Comment by malysps** [comment]
Unfortunately, seems like 22 size is no longer available in the upstream, and I don't know how to properly edit/resize SVG files, so I can't help creating these missing icons.

Easy workaround would be "creating" (exporting in GIMP) 22x22 PNG files instead of SVG. But I guess it's not a clean and desirable solution.

**Comment by ochosi** [comment]
You can help by creating a pull request with everything else then I'll take care of the missing 22px version

**Comment by malysps** [comment]
I've created pull request with those changes.
https://github.com/shimmerproject/elementary-xfce/pull/131
